main.cpp

```diff
@@ -13,75 +13,606 @@ What you create in this project will be used as the basis of Project 5 in the co
+    int numClassrooms = 80; 
+    //number of laboratories
+    int numLabs = 36; 
+    //number of professors
+    int numProfessors = 95; 
+    //amount of semestral income
+    float semIncome = 5'000'000; 
+    //classes taken per period
+    int classesPerSemester = 8362; 
+
+    struct Professor
+    {
+        int age = 58;
+        std::string profession = "Engineer";
+        std::string postgraduateStudies = "MsC";
+        std::string teachingRank = "Band 4";
+        int yearsExperience = 15;
+
+        float work(int research = 28, int teaching = 16);
+        float monthlyPay(float rent = 1800, 
+                         float food = 550, 
+                         float fun = 120,
+                         float other = 150);
+    };
+
+    struct Student
+    {
+        int age = 18;
+        std::string department = "Engineer";
+        std::string career = "AgroIndustrial Engineer";
+        std::string hobby = "Play Bass";
+        int semestralCredits = 9;
+
+        float weekstudyTime(int research = 28, int teaching = 16);
+        float monthlyExpenses(float food = 250, 
+        float fun = 220,
+        float other = 350);
+    };
+
+    //Teach students
+    float teachStudents(Professor Frank); 
+    //Do collaborate research 
+    int doCollaborateResearch(std::string organization = "UCLA", 
+                              std::string department = "Chemistry",
+                              std::string projectName = "influence of fertilizer on plant cultivation");
+    //Perform cultural activities
+    int performCulturalActivities(Student Peter, 
+                           std::string category = "Arts",
+                           std::string activity = "Museum Visit");
+    };
+
+/*
+Thing 3) Computer
+5 properties:
+    1) amount of used energy (float)
+    2) amount of required memory to run multiple programs (float)
+    3) amount of daily used disk space (float)
+    4) number of execution errors (int)
+    5) task executed per day (int)
+3 things it can do:
+    1) Execute Programs 
+    2) Save Information
+    3) Connect with other computers
+ */
+ 
+struct Computer
+{
+    //amount of used energy
+    float energyConsumption = 35.5f; 
+    //amount of required memory to run multiple programs
+    float requiredRAM = 6.9f; 
+    //amount of daily used disk space
+    float diskSpace = 89.5f; 
+    //number of execution errors
+    int execErrors = 48; 
+    //task executed per day
+    int execTask = 25; 
+
+    //Computer Hardware Specs
+    struct Hardware 
+    {
+      //[Gb] Total RAM
+      float RAM = 16;
+      //[Gb] Total Disk space
+      float disk = 1024; 
+      //[GHz] CPU Speed
+      float CPU = 4.5;
+      //[Gb] Total GPU 
+      float GPU = 16; 
+      //[Inches] Screen Size
+      float screen = 24;
+        
+      bool playGames(bool specs = true);
+      bool trainAI(std::string softwareReq = "Keras");
+      bool workAtOffice(std::string mostlyUsedTask = "Accounting");
+    };
```

Comment:
Fix this indentation.
Anything inside curly braces needs to be indented 4 spaces from the curly braces.
- [ ] revise